examples/energy-management-app/energy-management-common/energy-evse/src/ChargingTargetsMemMgr.cpp

```diff
@@ -114,51 +114,28 @@ CHIP_ERROR ChargingTargetsMemMgr::AllocAndCopy()
 
     return CHIP_NO_ERROR;
 }
-
 CHIP_ERROR
 ChargingTargetsMemMgr::AllocAndCopy(const DataModel::List<const Structs::ChargingTargetStruct::Type> & chargingTargets)
 {
-    // NOTE: ChargingTargetsMemMgr::PrepareDaySchedule() must be called as specified in the class comments in
```

Comment:
This is a good demonstration of the benefit we get being able to iterate `List<T>` & `DecodableList<T>` in the same way